src/test/java/com/rapid7/jul/LogentriesHandlerTest.java

```diff
@@ -0,0 +1,66 @@
+package com.rapid7.jul;
+
+
+import org.junit.After;
+import org.junit.Test;
+
+import java.util.logging.LogManager;
+import java.util.logging.Logger;
+
+import static junit.framework.TestCase.assertEquals;
+
+public class LogentriesHandlerTest {
+
+    @After
+    public void resetLogger() {
+        LogManager.getLogManager().reset();
+    }
+
+    @Test
+    public void singleLogger() throws Exception {
+        final String message = "single logger test";
+        final String token = "token-logentries-handler";
+        SocketChannelReceiver receiver = null;
+        try {
+            receiver = SocketChannelReceiver.createAndStartReceiver(10000);
```

Comment:
Rather than using static ports, can we do something like this?

```
   private int findFreePort(int port) {
        ServerSocket socket = null;
        try {
            socket = new ServerSocket(port);
            return port;
        } catch (IOException ex) {
            return findFreePort(port + 1);
        } finally {
            if (socket != null) {
                try {
                    socket.close();
                } catch (IOException e) {
                    // ignore
                }
            }
        }
    }
```

Different class, but you get the idea.

Reply:
the port number has to match what is set in the properties file.

Reply:
Fair point. I'm just wondering if there's a way we can be more protective against port collisions here.